systemvm/patches/debian/config/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -407,10 +410,10 @@ def fw_vpcrouter(self):
                             ])
 
         if self.get_type() in ["public"]:
-            self.fw.append(["nat", "front",
-                            "-A POSTROUTING -o %s -j SNAT --to-source %s" %
-                           (self.dev, self.address['public_ip'])
-                            ])
+            # self.fw.append(["nat", "front",
```

Comment:
The idea is to get his PR merged, then I will apply the fixes, remove the commented lines, test and create a new PR.

No, the changes above won't break any existing tests. The redundant VPC has just 1 test till now, which tests the offering and creation of rVPC - it was added by @afornie. @isoutham is adding the second test from scratch, so that won't break anything.

Cheers,
Wilder